src/renderer/lib/config.ts

```diff
@@ -39,6 +40,7 @@ const defaultConfig: WinboatConfigObj = {
     rdpArgs: [],
     disableAnimations: false,
     containerRuntime: ContainerRuntimes.DOCKER, // TODO: Ideally should be podman once we flesh out everything
```

Comment:
Put comment on line above

```suggestion
    // TODO: Ideally should be podman once we flesh out everything
    containerRuntime: ContainerRuntimes.DOCKER,
```